lib/nvmf/subsystem.c

```diff
@@ -51,6 +51,9 @@
 #define MODEL_NUMBER_DEFAULT "SPDK bdev Controller"
 #define NVMF_SUBSYSTEM_DEFAULT_NAMESPACES 32
 
+#define NVMF_CNTLID_MIN 1
+#define NVMF_CNTLID_MAX 0xFFEF
```

Comment:
I know it's mentioned below, but perhaps add a comment here indicating that values 0xfff0-0xffff are reserved.

Reply:
Done.